packages/pigeon/tool/test.dart

```diff
@@ -105,5 +107,5 @@ ${parser.usage}''');
     }
   }
 
-  await runTests(testsToRun);
+  await runTests(testsToRun, skipGen: true);
```

Comment:
Presumably this was supposed to use the flag?

Reply:
lol